CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+## 24 Jan 2018
+- Updated to mdc 0.28.0.
+- All types of drawers now share same DrawerHeader, DrawerContent.
+```
+
+```
+- ListItemIcon has renamed props, `start-detail` => `graphic` and `end-detail` => `meta`.
+- Button has `primary`/ `secondary` props to button.
```

Comment:
Maybe change the wording here to "Button has `primary` & `secondary` props for setting the theme color"... or something